modules/fulcrum.nix

```diff
@@ -0,0 +1,133 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  options.services.fulcrum = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable fulcrum, an Electrum server implemented in C++.
+
+        Compared to electrs, fulcrum has a 3x larger database size but
+        can serve arbitrary address queries instantly.
+
+        This module disables peering (a fulcrum-native feature for maintaining
+        a distributed list of electrum servers), but you can manually enable
+        it via option `extraConfig`.
```

Comment:
Will setting it via `extraConfig` override the `peering = false` defined earlier in the config?

Reply:
I've added it as an `extraConfig` example.

Reply:
Ok, good